app/models/ems_refresh/save_inventory.rb

```diff
@@ -191,6 +191,8 @@ def save_hardware_inventory(parent, hash)
   def save_guest_devices_inventory(hardware, hashes)
     return if hashes.nil?
 
+    hashes.flatten!
+
```

Comment:
@djberg96 
Can we flatten the array locally in the SCVMM parser instead? It would be this line:
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb#L412